tests/compiler.js

```diff
@@ -1125,5 +1125,35 @@
                   'may the forth be with you');
             finish(done);
         });
+        
+
+        it('should thrown an error', function(done) {
```

Comment:
Can we get a more specific description of the previously-broken case here? Like `it('should throw an error when including a file that calls an undefined macro')`?